frontend/src/components/BlogCard.tsx

```diff
@@ -36,7 +39,20 @@ const BlogCard = ({ author, title, content, publishedDate, id }: BlogCardProps)
 				<div className="tracking-wide py-4 text-slate-600">
 					<ReactQuill value={quillContent} readOnly={true} theme={"bubble"} />
 				</div>
-				<div className="text-gray-600">{Math.ceil(content.length / 300)} min read</div>
+				<div className="flex justify-between">
+					<div className="flex flex-wrap">
+					{blog.tagsOnPost.slice(0,2).map((tagWrapper) => {
+						return (
+							<div className="px-1" key={tagWrapper.tag.id}> 
+								<div className="bg-gray-200 p-2 flex justify-center rounded-3xl text-sm">
+									{tagWrapper.tag.tagName}
+								</div>
+							</div>
+						)
+					})}
+					</div>
+					<div className="text-gray-600 pt-1">{Math.ceil(content.length / 300)} min read</div>
+				</div>
```

Comment:
The BlogCard is out of scope of this ticket. Please rollback the changes for this component.